### DISCUSSION TITLE
[Feature Request] Fingerprint 2FA authentication option for GitHub Mobile

### DISCUSSION THREAD

**Discussion by d-kleine** [community/community#119790] [Mobile]
### Select Topic Area

Product Feedback

### Body

When signing in to GitHub with your browser using [two-factor authentication (2FA)](https://docs.github.com/de/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication), you can verify your identity with your smartphone/tablet. When signed in, the browser will then display two digits, the new sign in request then needs to be confirmed by typing in those digits in the GitHub Mobile app:
![Sign in 2FA](https://i0.wp.com/user-images.githubusercontent.com/1666363/172505068-fcd80033-cc79-4d9a-92f4-ced356f0f60e.png?ssl=1)

Typing in those two random numbers can sometimes be exhausting. Would it be possible to add a fingerprint authentication option to the GitHub Mobile app to confirm the sign in when using 2FA?

**Comment by stevepopovich** [comment]
Hi @d-kleine thanks for the feedback. I have filed a feature request internally for this. Thank you!

**Reply by d-kleine** [reply]
Thank you!

**Reply by d-kleine** [reply]
@stevepopovich Any update on this feature request yet?

**Reply by stevepopovich** [reply]
We are still investigating this internally!

**Reply by d-kleine** [reply]
Okay, thank you for the update!

**Reply by Morton22** [reply]
This person trying to hack my project management 💔💔

**Reply by Morton22** [reply]
I want you to cancel the order 2F on my browser to enable me to log in

**Reply by Neels4444** [reply]
> We are still investigating this internally!

**Comment by ghost** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> When signing in to GitHub with your browser using [two-factor authentication (2FA)](https://docs.github.com/de/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication), you can verify your identity with your smartphone/tablet. When signed in, the browser will then display two digits, the new sign in request then needs to be confirmed by typing in those digits in the GitHub Mobile app:
> 
> ![Sign in 2FA](https://i0.wp.com/user-images.githubusercontent.com/1666363/172505068-fcd80033-cc79-4d9a-92f4-ced356f0f60e.png?ssl=1)
> 
> 
> 
> Typing in those two random numbers can sometimes be exhausting. Would it be possible to add a fingerprint authentication option to the GitHub Mobile app to confirm the sign in when using 2FA?

**Reply by Morton22** [reply]
13

**Comment by Thrumanshow** [comment]
Una 🔑 es lo mejor 🚀
